test/polyfills.js

```diff
@@ -12,6 +12,15 @@ import 'core-js/fn/string/code-point-at';
 import 'core-js/fn/string/from-code-point';
 import 'core-js/fn/string/repeat';
 
+Object.defineProperty(Array.prototype, '-1', {
```

Comment:
This is great! ❤️

Reply:
Agree, really love this for catching perf issues :+1: